### PR TITLE
🎉 New Source - Secoda [low-code cdk]

### PR DESCRIPTION
**Pull request by haithem-souala**
## What
New source connector https://docs.secoda.co/secoda-api

## How
*Describe the solution*

## Streams
1. Collections
2. Tables
3. Terms

## 🚨 User Impact 🚨
N/A

## Pre-merge Checklist
Expand the relevant checklist and delete the others.

<details><summary><strong>New Connector</strong></summary>

### Community member or Airbyter

- [x] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret`
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [x] Documentation updated
    - [x] Connector's `README.md`
    - [x] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [x] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [x] `docs/integrations/README.md`
    - [x] `airbyte-integrations/builds.md`
- [x] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
- [ ] After the connector is published, connector added to connector index as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
- [ ] Seed specs have been re-generated by building the platform and committing the changes to the seed spec files, as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)

</details>

## Tests

<details><summary><strong>Unit</strong></summary>

*Put your unit tests output here.*

</details>

<details><summary><strong>Integration</strong></summary>

*Put your integration tests output here.*

</details>

<details><summary><strong>Acceptance</strong></summary>

Capture d’écran 2022-10-24 à 17 40 28

</details>

### CODE REVIEW

**Comment by haithem-souala** [comment]
Capture d’écran 2022-10-24 à 17 40 28

**Comment by marcosmarxm** [comment]
Thanks for the contribution! Someone will review it during this week. If you have any questions please create comments so the reviewer can focus on those points.

File: airbyte-integrations/connectors/source-gutendex/source_gutendex/gutendex.yaml [hunk 1]
```diff
@@ -25,13 +25,10 @@ definitions:
...
       pagination_strategy:
         type: "PageIncrement"
-        page_size: 32
+        page_size: 100
       page_token_option:
         inject_into: "request_parameter"
         field_name: "page"
-      page_size_option:
-        inject_into: "body_data"
-        field_name: "page_size"
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-gutendex/source_gutendex/gutendex.yaml:34]: Why this change?

File: airbyte-integrations/connectors/source-secoda/integration_tests/catalog.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{
+  "streams": [
+    {
+      "name": "TODO fix this file",
+      "supported_sync_modes": ["full_refresh", "incremental"],
+      "source_defined_cursor": true,
+      "default_cursor_field": "column1",
+      "json_schema": {
+        "$schema": "http://json-schema.org/draft-07/schema#",
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-secoda/integration_tests/catalog.json:9]: Remove this file.

File: airbyte-integrations/connectors/source-secoda/source_secoda/schemas/TODO.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# TODO: Define your stream schemas
+Your connector must describe the schema of each stream it can output using [JSONSchema](https://json-schema.org). 
+
+You can describe the schema of your streams using one `.json` file per stream.
+ 
+## Static schemas
+From the `secoda.yaml` configuration file, you read the `.json` files in the `schemas/` directory. You can refer to a schema in your configuration file using the `schema_loader` component's `file_path` field. For example:
+```
+schema_loader:
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-secoda/source_secoda/schemas/TODO.md:9]: Remove this file.

File: airbyte-integrations/connectors/source-secoda/source_secoda/secoda.yaml [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+      path: "/dictionary/terms/"
+  collections_stream:
+    $ref: "*ref(definitions.base_stream)"
+    retriever:
+      record_selector:
+        $ref: "*ref(definitions.selector)"
+      paginator:
+        $ref: "*ref(definitions.cursor_paginator)"
+      requester:
+        $ref: "*ref(definitions.requester)"
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-secoda/source_secoda/secoda.yaml:70]: All streams use the same ref, why you don't add those to the base stream and create it using the ref?

File: airbyte-integrations/connectors/source-secoda/source_secoda/spec.yaml [hunk 5]
```diff
@@ -0,0 +1,17 @@
+documentationUrl: https://docsurl.com
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-secoda/source_secoda/spec.yaml:1]: update this to correct secoda airbye doc url

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

File: airbyte-integrations/connectors/source-gutendex/source_gutendex/gutendex.yaml (hunk 1 above)
Comment (haithem-souala) [reply airbyte-integrations/connectors/source-gutendex/source_gutendex/gutendex.yaml]: Opss, my bad! i revert changes.

File: airbyte-integrations/connectors/source-secoda/source_secoda/secoda.yaml (hunk 4 above)
Comment (haithem-souala) [reply airbyte-integrations/connectors/source-secoda/source_secoda/secoda.yaml]: Indeed.

File: airbyte-integrations/connectors/source-secoda/source_secoda/secoda.yaml [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+    page_token_option:
+      inject_into: path
+    page_size_option:
+      inject_into: body_data
+      field_name: "page_size"
+    pagination_strategy:
+      type: "CursorPagination"
+      cursor_value: "{{ response.links.next }}"
+      stop_condition: "{{ response.links.next is none}}"
+      page_size: 1
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-secoda/source_secoda/secoda.yaml:25]: @haithem-souala the page size is one? this is correct?

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
last comment and ready to 🚀

**Comment by marcosmarxm** [comment]
Hello! Your PR is approved but didn't have the time to publish and merge it this week. As you can check in [Chris' comment](https://airbytehq.slack.com/archives/C0450PCSDB2/p1667430678866629) all PRs submitted before 2-nov are eligible to win the prize. I'll be out of the office on Friday and return Monday to start publishing your contribution. Any question you can send a message in `#hacktoberfest-2022` channel in Airbyte's Slack.

Have a good weekend and thank you for this amazing contribution for Hacktoberfest 🎉

**Review by marcosmarxm — APPROVED** [review]
Thanks @haithem-souala
